HomeWorks/HW.05.Task2/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace HW05.Task2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("vvod");
+            int a = Convert.ToInt32(Console.ReadLine());
+            string emp = null;
+            for (int i = 1; i <= a; i++)
+            {
+                emp = emp + " ";
+                string str = null;
+                for (int j = i; j <= a; j++)
+                {
+                    str = str + " " + Convert.ToString(i);
+                }
+                Console.WriteLine(emp + str);
+            }
+        }
+    }
+}
```

Comment:
it works but with some issues, when the number >= 10

![image](https://user-images.githubusercontent.com/3023032/109002456-4a258e00-76b7-11eb-8bdf-ee53e492ab33.png)

Reply:
should be `0` instead of 10
should be `1` instead of 11

in general, it should be: `% 10`